pkg/cloud-controller-manager/loadBalancer.go

```diff
@@ -260,6 +260,8 @@ func extractHealthCheck(svc *v1.Service) (*lbv1.HeathCheck, error) {
 	}
 	if port != nil {
 		healthCheck.Port = *port
+	} else {
+		return nil, nil
```

Comment:
not sure set the line #248 `HealthCheck: nil` will be a good practice, ideally we should skip to setting the value when it is nil.

Reply:
It's not necessary to judge whether Healthcheck is nil here.